objects/app/apps.go

```diff
@@ -92,10 +92,10 @@ func (a *appsCmd) list(c *cli.Context) error {
 }
 
 func appWithFlags(c *cli.Context, app *modelsv2.App) {
-	if app.SyslogURL == "" {
+	if len(c.String("syslog-url")) > 0 {
 		app.SyslogURL = c.String("syslog-url")
 	}
-	if len(app.Config) == 0 {
+	if len(c.StringSlice("config")) > 0 {
```

Comment:
does string slice allow specifying this multiple times? because that would be awesome.